### PR TITLE
add missing npm module for system message markdown

### PR DESCRIPTION
**Pull request by mithunjohny**
## **Thank you for your contribution. Before submitting this PR, please include:**

https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3857360.

### Description
anchor tag in system messages don't have the required security attributes rel="noopener noreferrer", Space between the system messages are high

## Solution Proposed
Fixed missing npm module simple-update-in dependency to in htmlTextMiddleWare, also added a css to remove the default p tag margin

## Test cases and evidence
image

image

### Sanity Tests
-   [x] You have tested all changes in Popout mode
-   [x] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/package.json [hunk 1]
```diff
@@ -85,14 +85,15 @@
...
-    "slack-markdown-it": "^1.0.5"
+    "slack-markdown-it": "^1.0.5",
+    "simple-update-in":"2.2.0"
   },
   "scripts": {
     "storybook": "start-storybook -p 6006",
     "compose-storybook": "start-storybook -c stories/.storybook -p 9009",
     "build-composite-storybook": "build-storybook -c stories/.storybook -o storybook-build",
     "build-storybook": "build-storybook",
-    "build": "yarn lint && yarn build:esm && yarn build:cjs && tsc",
```
Comment (elopezanaya) [inline chat-widget/package.json:95]: why was needed to remove yarn lint?

File: chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/htmlTextMiddleware.ts [hunk 2]
```diff
@@ -66,7 +67,7 @@ const processHTMLText = (action: IWebChatAction, text: string): IWebChatAction =
             }
 
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            action = (window as any).simpleUpdateIn(action, [Constants.payload, Constants.activity, Constants.text], () => htmlNode.innerHTML);
```
Comment (elopezanaya) [inline chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/htmlTextMiddleware.ts:69]: we need to test this using CRM Livechat, also using popout

**Review by elopezanaya — CHANGES_REQUESTED** [review]

File: chat-widget/package.json (hunk 1 above)
Comment (mithunjohny) [reply chat-widget/package.json:95]: my bad, mac issue , forgot to put it back before pushing

File: chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/htmlTextMiddleware.ts (hunk 2 above)
Comment (mithunjohny) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/htmlTextMiddleware.ts:69]: yes tested popout mode too

**Review by elopezanaya — APPROVED** [review]

**Pull request merged**